acceptance/config/nodes/huaweios-6-powerpc.yaml

```diff
@@ -0,0 +1,13 @@
+---
+HOSTS:
+  huaweios-6-powerpc:
+    roles:
+      - agent
+    platform: huaweios-6-powerpc
+    hypervisor: none
+    ip: huawei-ce6850-2-debian-vm-eth0.ops.puppetlabs.net
```

Comment:
The above is incorrect. The ip is set to the hostname. I believe it should look like the following:
```
ip: 10.0.7.26
vmhostname: huawei-ce6850-2-debian-vm-eth0.ops.puppetlabs.net
```